### DISCUSSION TITLE
dvd menu support

### DISCUSSION THREAD

**Discussion by metalfoxdev** [smplayer-dev/smplayer#286] [Ideas]
Hi there, I found this app a while ago and I love it so much. It plays everything and has excellent yt support with resolutions of over 4K (compared to VLC's 720p). The other day i tried playing a DVD with it. (I was trying to navigate to an extra on the disc) but I ran into an issue, SMplayer would not let me select anything with my mouse cursor on the menu. is there any support for navigation of dvd menus coming any time soon?

**Comment by smplayer-dev** [comment]
DVD support was removed some time ago because mpv removed it. You can select the title to play using the menu Browse -> Title.

**Reply by metalfoxdev** [reply]
Thank you! 🙂

**Reply by cipricus** [reply]
> DVD support was removed some time ago because mpv removed it. You can select the title to play using the menu Browse -> Title.

I think that depends on the dvd disc - but **is there a way to just play the main video like mpv does with `mpv dvd://`**? 

On a dvd I'm trying to play there is no title under Browse, while the option to open dvd folder just brings the dvd menus interface with unclickable links.

**Comment by bendipa** [comment]
Thanks from me as well. It's a real pity SMplayer no longer supports DVD menu titles. It's actually Browse -> Chapter that I find works for starting DVDs from the beginning of a movie. _Title_ shows as empty.

So now all I need is to find a way how to switch on subtitles. The conventional _Subtitles_ tab does not  switch on embedded subtitles either. Any ideas how to proceed on that?

**Reply by smplayer-dev** [reply]
Could you send the mpv log (menu View) when playing the DVD?
